server/src/test/java/com/cloud/network/vpn/RemoteAccessVpnManagerImplTest.java

```diff
@@ -0,0 +1,224 @@
+/*
+ * Copyright 2021 The Apache Software Foundation.
```

Comment:
@GutoVeronezi as far as I know (please correct me if I am wrong), it should not contain blocks such as `Copyright <YEAR> The Apache Software Foundation`.

At least this is not the policy for Apache source code. If someone of a random project/organization wants to adopt the Apache license 2.0 then it can add the Copyright. Vide:https://www.apache.org/licenses/LICENSE-2.0


BTW I am following the Apache legal documentation for the source code header on [Source File Headers for Code Developed at the ASF](https://www.apache.org/legal/src-headers.html#headers). You can find it here: https://www.apache.org/legal/src-headers.html#headers

Reply:
I don't think this is a problem @GabrielBrascher , @GutoVeronezi is donating to apache on his own accord.

Reply:
@DaanHoogland I might be wrong on this one.

Do you understand that just committers and PMCs need to add source headers accordingly to https://www.apache.org/legal/src-headers.html#headers but contributors outside Apache can do it as https://www.apache.org/licenses/LICENSE-2.0?

Reply:
@GabrielBrascher done.